libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectUtils.kt

```diff
@@ -0,0 +1,47 @@
+@file:JvmName("RoadObjectUtils")
+
+package com.mapbox.navigation.base.trip.model.roadobject
+
+import androidx.annotation.IntRange
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.core.constants.Constants
+import com.mapbox.geojson.LineString
+import com.mapbox.navigation.base.utils.ifNonNull
+
+private const val PRECISION_RANGE_START = 5L
+private const val PRECISION_RANGE_END = 6L
```

Comment:
```suggestion
private const val PRECISION_RANGE_START = 5
private const val PRECISION_RANGE_END = 6
```

Reply:
must be `long`. With `int` const doesn't compile
<img width="627" alt="Screenshot 2021-03-16 at 09 41 04" src="https://user-images.githubusercontent.com/14282440/111266584-cdddf500-863b-11eb-9883-d50593fba2fa.png">